uniflow/flow/config.py

```diff
@@ -241,11 +242,184 @@ class RaterClassificationOpenAIGPT4Config(RaterConfig):
 
 
 @dataclass
-class RaterForGeneratedAnswerConfig(RaterConfig):
-    """Rater classification Config Class."""
+class RaterForClassificationOpenAIGPT3p5Config(RaterConfig):
+    """
+    The configuration primarily focuses on setting up the parameters for utilizing GPT-3.5 to evaluate the
+    correctness of answers in relation to given questions and contexts.
 
-    flow_name: str = "RaterFlow"
-    model_config: ModelConfig = field(default_factory=OpenAIModelConfig())
+    Attributes:
+        flow_name (str): Name of the rating flow, default is "RaterFlow".
+        model_config (ModelConfig): Configuration for the GPT-3.5 model. Includes model name ("gpt-3.5-turbo-1106"),\
+                                    the server ("OpenAIModelServer"), number of calls (3), temperature (0.9),\
+                                    and the response format (plain text).
+        label2score (Dict[str, float]): Mapping of labels to scores, default is {"Yes": 1.0, "No": 0.0}.
+        guided_prompt_template (GuidedPrompt): Template for guided prompts used in rating. Includes instructions\
+                                               for rating, along with examples that detail the context, question,\
+                                               answer, label, and explanation for each case.
+    """
+
+    model_config: ModelConfig = field(
+        default_factory=lambda: OpenAIModelConfig(
+            model_name="gpt-3.5-turbo-1106",
+            model_server="OpenAIModelServer",
+            num_call=3,
+            temperature=0.9,
```

Comment:
Similar to the above comment, empirical result gives better performance for using deterministic mode to use LLM as a rater, you should consider to change num_cal=1 and temperature=0. This will also help mitigate the problem you proposed in the slack channel regarding tie cases.